test/unittest/unittest_nntrainer_graph.cpp

```diff
@@ -0,0 +1,138 @@
+// SPDX-License-Identifier: Apache-2.0
+/**
+ * Copyright (C) 2020 Jijoong Moon <jijoong.moon@samsung.com>
+ *
+ * @file unittest_nntrainer_graph.cpp
+ * @date 29 Oct 2020
+ * @brief NNTrainer graph test.
+ * @see	https://github.com/nnstreamer/nntrainer
+ * @author Jijoong Moon <jijoong.moon@samsung.com>
+ * @bug No known bugs except for NYI items
+ */
+
+#include <gtest/gtest.h>
+#include <neuralnet.h>
+
+#include "nntrainer_test_util.h"
+
+namespace initest {
+typedef enum {
+  LOAD = 1 << 0, /**< should fail at load */
+  INIT = 1 << 1, /**< should fail at init */
+} IniFailAt;
+};
+
+class nntrainerGraphTest
```

Comment:
Any reason for the duplicated test? I think this can go into `modelfile` test

Reply:
More test cases are coming which related with graph like initialize, compiling, forwarding and so on.

Reply:
If the purpose is to use Ini, I think it should use `IniTestWrapper`. This test checks if model file load & compiles and initiate without any problem which is not related to graph

Reply:
This is just skeleton at his stage. I will add more test cases accordingly.